.changeset/nine-ladybugs-hide.md

```diff
@@ -0,0 +1,7 @@
+---
+"@utrecht/component-library-react": major
+---
+
+added html heading component to component library react
+added html heading test file to component library react
+refactor heading in accordion
```

Comment:
Test files en refactorings hoeven niet genoemd te worden in de changeset, die hebben geen impact op gebruikers.

Wat is de breaking change voor gebruikers dat je voor major hebt gekozen?